src/sentry/shared_integrations/client/base.py

```diff
@@ -481,9 +499,16 @@ def disable_integration(self, buffer) -> None:
         )
 
         if (
-            features.has("organizations:slack-fatal-disable-on-broken", org)
-            and rpc_integration.provider == "slack"
-        ) and buffer.is_integration_fatal_broken():
+            (
+                features.has("organizations:slack-fatal-disable-on-broken", org)
```

Comment:
Is this from the other PR too?

Reply:
Yeah I think it just got reformatted since I added the github flag.